weave/Scripts/Utils/GDScriptHelper.cs

```diff
@@ -0,0 +1,13 @@
+using Godot;
+
+namespace Weave;
+
+public static class GDScriptHelper
+{
+    public static ImageTexture GenerateQRCodeFromString(string str)
```

Comment:
I mentioned this in the last PR, but perhaps ""removing"" the gdscript dependency with a IQrCodeGenerator